supervisor/resolution/free_space.py

```diff
@@ -47,7 +50,10 @@ def check_free_space(self) -> None:
             suggestions.append(SuggestionType.CLEAR_FULL_SNAPSHOT)
 
         self.sys_resolution.create_issue(
-            IssueType.FREE_SPACE, ContextType.SYSTEM, suggestions=suggestions
+            IssueType.FREE_SPACE,
+            ContextType.SYSTEM,
+            suggestions=suggestions,
+            reference=str(free_space),
```

Comment:
The reference is part of ContextType for id. Please make a new field like additional